### PR TITLE
[ROOT-10302] Move glew to ROOT builtins (part 2)

### PR DESCRIPTION
**Pull request by oshadura**
Replacement of https://github.com/root-project/root/pull/4095

### CODE REVIEW

**Comment by oshadura** [comment]
@Axel-Naumann  can you review? I would like to land it in 6.22...

File: builtins/glew/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+set(GLEW_FOUND TRUE CACHE BOOL "" FORCE)
+
+set(GLEW_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/inc CACHE INTERNAL "" FORCE)
+set(GLEW_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/inc CACHE INTERNAL "" FORCE)
+
+if(APPLE)
+  if(NOT cocoa)
+    set(GLEW_DEFINITIONS -DGLEW_APPLE_GLX)
+  endif()
+endif()
```
Comment (Axel-Naumann) [inline builtins/glew/CMakeLists.txt:34]:
```suggestion
if(APPLE AND NOT cocoa)
  set(GLEW_DEFINITIONS -DGLEW_APPLE_GLX)
endif()
```

File: builtins/glew/inc/GL/eglew.h [hunk 2]
```diff
@@ -0,0 +1,2618 @@
+/*
```
Comment (Axel-Naumann) [inline builtins/glew/inc/GL/eglew.h:1]: Why do we need this header now - why did we not need it before?

File: graf3d/rglew/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+# This package is only needed for user backward compatibility! (TGLIncludes.h)
+
+add_library(RGlew INTERFACE)
+target_include_directories(RGlew INTERFACE inc/)
+target_link_libraries(RGlew INTERFACE GLEW)
+#target_include_directories(RGlew INTERFACE $<BUILD_INTERFACE:${GLEW_INCLUDE_DIR}>)
+
+# We still need to install TGLIncludes.h into include/
+ROOT_INSTALL_HEADERS()
```
Comment (Axel-Naumann) [inline graf3d/rglew/CMakeLists.txt:19]: \n at EOL please!

File: graf3d/gl/CMakeLists.txt [hunk 4]
```diff
@@ -230,6 +230,10 @@ else()
...
 
 
+if(MSVC)
+  target_compile_definitions (RGL PRIVATE "GLEW_STATIC")
+endif()
+
 if(MACOSX_GLU_DEPRECATED)
   target_compile_options(RGL PRIVATE -Wno-deprecated-declarations)
-endif()
+endif()
```
Comment (Axel-Naumann) [inline graf3d/gl/CMakeLists.txt:239]: \n at EOL please

**Pull request merged**